serverest/tests/commons/Commons.robot

```diff
@@ -0,0 +1,23 @@
+***Keywords***
+Create Users
+    [Documentation]     Used to create users.
+    [Arguments]         ${nome}     ${email}     ${password}     ${administrador}
+
+    &{USER}                             Create Dictionary
+    ...                                 nome= ${nome}
+    ...                                 email=${email} 
+    ...                                 password=${password}3
+    ...                                 administrador=${administrador}
+
+    Log                                 ${USER}
+    Set Test Variable                   ${USER}
+
+
+Get Token
+    [Documentation]     Used to get a token.
+    Create Users                        Phao  phao@qa.com  123  True
+    Login                               phao@qa.com  123                              
```

Comment:
@RafaelFelice I said we can't set a parameter on this KW, remember?

For example: phao